plotly/figure_factory/_bullet.py

```diff
@@ -0,0 +1,257 @@
+from __future__ import absolute_import
+
+import math
+
+from plotly import colors, exceptions, optional_imports
+from plotly.figure_factory import utils
+
+import plotly
+import plotly.graph_objs as go
+
+pd = optional_imports.get_module('pandas')
+
+VALID_KEYS = ['title', 'subtitle', 'ranges', 'measures', 'markers']
+
+
+def _bullet(df, as_rows, marker_size, marker_symbol, range_colors,
+            measure_colors, subplot_spacing):
+    num_of_lanes = len(df)
+    num_of_rows = num_of_lanes if as_rows else 1
+    num_of_cols = 1 if as_rows else num_of_lanes
+    if not subplot_spacing:
+        subplot_spacing = 1./num_of_lanes
+    fig = plotly.tools.make_subplots(
+        num_of_rows, num_of_cols, print_grid=False,
+        horizontal_spacing=subplot_spacing,
+        vertical_spacing=subplot_spacing
```

Comment:
does it make sense to provide `horizontal_spacing` and `vertical_spacing` as input arguments to `ff.make_bullet` instead? that way it's consistent with `make_subplots`

Reply:
Yes it does!